src/components/backend-ai-session-list.ts

```diff
@@ -1006,14 +1006,14 @@ export default class BackendAiSessionList extends BackendAIPage {
     if (mounts.length > 1) {
       mounts.map((key, index) => {
         if (index > 0) {
-          let item = document.createElement('mwc-list-item');
-          item.innerHTML = key[0];
```

Comment:
Use of simpler name `item` here would be okay since `item`'s scope is limited to the `if` block at line 1008. But, I think it's okay.